src/transforms/levels.jl

```diff
@@ -0,0 +1,72 @@
+"""
+    Levels(:a => ["yes,"no"])
+
+Return a copy of the table with specified levels and orders for categorical columns
+allowing only changing the order of the column
+
+# Examples
+
+```julia
+Levels(:a => ["yes,"no"], :c => [1,2,4], :d = ["a","b","c"])
+Levels("a" => ["yes","no"], "c" => [1,2,4], ordered= ["a","c"])
+Levels(:a => ["yes","no"], :c => [1,2,4], :d => [1,23,5,7] ,ordered = [:a,:b,:c])
+```
+"""
+
+@kwdef struct Levels{K} <: Stateless
+  levelspec::K
+  ordered::AbstractVector{Symbol}
+end
```

Comment:
Please remove the @kwdef dependency and use another type for the ordered field that is a concrete type, for example Vector{Symbol}